.github/workflows/continuous-integration-unit-php.yml

```diff
@@ -24,12 +24,25 @@ jobs:
     continue-on-error: ${{ matrix.experimental == true }}
     strategy:
       matrix:
-        php: ['7.4', '7.0', '5.6']
+        php: ['7.3', '7.0', '5.6']
         wp: ['latest']
+        multisite: [false]
+        coverage: [false]
         include:
           - php: '8.0'
             wp: 'trunk'
+            multisite: false
+            coverage: false
             experimental: true
+          - php: '7.4'
+            wp: 'latest'
+            multisite: true
+            coverage: false
```

Comment:
Don't we also need to run coverage against multisite? Otherwise multisite-only functionality won't be tracked.

Reply:
Yeah, I thought that myself. Should we just run the code coverage once or twice? One with and one without or both with?

Reply:
Twice for accurate results. Once with multisite, once without.

Reply:
Fixed.